src/adaptors/amqp/connection_manager.c

```diff
@@ -369,7 +363,7 @@ QD_EXPORT void qd_connection_manager_start(qd_dispatch_t *qd)
     }
 
     while (ctor_config) {
-        qd_connector_config_connect(ctor_config);
+        qd_connector_config_activate(ctor_config);
         ctor_config = DEQ_NEXT(ctor_config);
     }
 
```

Comment:
Can we please remove this `first_start` static variable as part of this change even though this PR has nothing to do with it? I don't understand why it is there

Reply:
The first_start var is a bit weird, but it seems to be related to detecting listener failures (and setting the listener->exit_on_error flag).  I'm not sure of the ramifications of removing that flag.  Since I've got to deal with listener lifecycle later (see Issue #1753) I'd recommend we hold off until then.

Reply:
`first_start` is set to true at the beginning of the function and never modified to false until the last line of the function which seems weird. That can be dealt with later. Thanks.